deploy/kubernetes/deploy-driver.sh

```diff
@@ -47,19 +69,30 @@ function check_service_account()
 	fi
 }
 
-check_service_account
+# Install kustomize if needed
+${PKGDIR}/deploy/kubernetes/install-kustomize.sh
```

Comment:
should this instead be a function in `common.sh` called `ensureKustomize` or something. Then you can run that function at the start of any script that requires Kustomize. We aren't guaranteed that anyone will run all the scripts, they could mix and match